controller/pages/api/util/testmanager.ts

```diff
@@ -1218,7 +1222,7 @@ export abstract class TestManager {
 
         // Pass pewpew version legacy/scripting
         const validateLegacyOnly = getValidateLegacyOnly(version);
```

Comment:
The await needs to be on this line, not the one below. You can catch this easier by changing it to a fixed type
```
const validateLegacyOnly: string | undefined = getValidateLegacyOnly(version);
```
and you'll get a better error and realize you need to do `await getValidateLegacyOnly(version);`

Reply:
Thanks